getting-started/bare-metal/about.md

```diff
@@ -1,56 +1,56 @@
 ---
 title: About non-cluster hosts
-description: Secure hosts not in a cluster by installing Calico with networking and/or networking policy enabled.
+description: Secure hosts not in a cluster by installing Calico with networking and/or networking policy.
 canonical_url: '/getting-started/bare-metal/about'
 ---
 
 ### Big picture
 
-Secure hosts not in a cluster by installing {{site.prodname}} with networking and/or networking policy enabled.
+Secure hosts by installing {{site.prodname}} for networking and/or networking policy.
```

Comment:
don't you need to specify "not in a cluster?"

Reply:
I changed it to match.